live/index.php

```diff
@@ -88,51 +88,53 @@ function gtag() { dataLayer.push(arguments); }
     <div id="header"></div>
 
     <div class="header_padding" id="live-stream-wrapper">
-        <div id="live-objects" class="center">
-            <h1><span style="color: red; font-weight: bold;">LIVE</span> AT CORNERSTONE</h1>
-            <h2>9:00am | 11:00am | 6:00pm ET</h2>
-            <div id="live-iframe">
-                <iframe id="live-stream" class="drop-shadow"
-                    src='https://www.youtube.com/embed/live_stream?channel=<?php echo $channelId ?>' frameborder="0"
+        <h1><span style="color: red; font-weight: bold;">LIVE</span> AT CORNERSTONE</h1>
+        <h2>9:00am | 11:00am | 6:00pm ET</h2>
+        <div id="live-iframe">
+            <div id="resize-wrapper">
+                <iframe class="drop-shadow"
+                    src='https://www.youtube.com/embed/live_stream?channel=UCSJ4gkVC6NrvII8umztf0Ow' frameborder="0"
                     allow="autoplay; accelerometer; encrypted-media; gyroscope; picture-in-picture"
                     allowfullscreen></iframe>
-                <iframe id="live-chat" frameborder="0">
-                </iframe>
             </div>
-            <div id="stream-offline">
-                <img src="/ref/pages/live_page/offline.png">
-                <h2 id="error-message">Stream Offline</h2>
-                <div id="error-description">
-                    <b>Looking for service? Check back at these times:</b>
-                    <br>Sunday Services 9:00am | 11:00am | 6:00pm ET
-                    <br>Daily Prayer at 12:00pm ET
-                </div>
-            </div>
-            <div id="countdown">
-                <div id="label-text">Next service starts in</div>
-                <div id="clock"></div>
-            </div>
-            <div id="button-group">
-                <a href="/give/" target="_blank" class="button--white">Give Online</a>
-                <a href="/events/" target="_blank" class="button--white">Our Events</a>
-                <a href="/about/" target="_blank" class="button--white">About Us</a>
+            <iframe id="live-chat" frameborder="0" src="https://www.youtube.com/live_chat?v=DWcJFNfaw9c&embed_domain=localhost" style="display: static"></iframe>
```

Comment:
Should the src be empty since it is being fetch and replaced by the PHP script on load?

Reply:
Hm, guess that was left there from tests. Yeah, the PHP does overwrite it, but it should be empty. Committed fix